WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
Absolutely not. Antinomians were among them ost hated groups in all of Christian heresies, to the point that Christ's Disciples and Apostles _themselves_ dedicated whole chapters to fighting the threat they posed. In addition, they are allowed to practice incest, and all carnal sin, as long as they ask for forgiveness. They are seen as savage by other Christians.

Besides, the opinion is balanced: all Antinomians do not have this penalty against each other.